lib/routes/account.js

```diff
@@ -888,6 +892,21 @@ module.exports = function (
       handler: function (request, reply) {
         log.begin('Account.RecoveryEmailStatus', request)
         var sessionToken = request.auth.credentials
+        // Desktop browsers poll this endpoint to check whether
+        // the account is verified, generating a *lot* of traffic.
+        // If the account remains unverified after too long,
+        // try to slow-walk their polling requests with 429s
+        // to reduce traffic.
+        var age = Date.now() - sessionToken.createdAt
+        log.histogram('recoveryEmailStatus.sessionAge', age / 1000)
```

Comment:
@dannycoates warned that we should not log here because this endpoint is heavy on traffic.